aspnetcore/security/authentication/community-auth.md

```diff
@@ -0,0 +1,14 @@
+# Community OSS Auth options
+
+When developing ASP.NET Core applications you can use [ASP.NET Core Identity](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/identity) to secure your web applications. You can also replace that with another auth provider. This page contains the known open source auth options for ASP.NET Core. This page will be periodically updated as new providers become available.
+
+# OSS Auth Providers
+
+The list below is sorted alphabetically sorted.
+
+| Name                      | Description | Link |
+|:--------------------------|------------:|------------------------------------------------:|
+| Identity Server 4 | Open Source OpenID Connect and OAuth 2.0 framework for .NET | https://identityserver.io/ |
+| OpenIdDict | Easy-to-use OpenID Connect server for ASP.NET Core | https://github.com/openiddict/openiddict-core |
+
+To get your provider added here [edit this page](INSERT-EDIT-THIS-PAGE-LINK-HERE).
```

Comment:
Use https://github.com/aspnet/Docs/edit/master/aspnetcore/security/authentication/community.md for the link